dev/local/install.sh

```diff
@@ -1,9 +1,26 @@
 #!/bin/bash
+# Bootstrap delphi-epidata
+#
+# Downloads the repos needed for local delphi-epidata development into current dir 
+# and provides a Makefile with Docker control commands.
 
-mkdir -p driver/repos/delphi driver-logs/delphi_database_epidata driver-logs/delphi_web_epidata
-cd driver/repos/delphi
+mkdir -p repos/delphi
+cd repos/delphi
 git clone https://github.com/cmu-delphi/operations
 git clone https://github.com/cmu-delphi/delphi-epidata
 git clone https://github.com/cmu-delphi/utils
+git clone https://github.com/cmu-delphi/flu-contest
+git clone https://github.com/cmu-delphi/nowcast
+git clone https://github.com/cmu-delphi/github-deploy-repo
```

Comment:
Where are these used?

Reply:
[Here](https://github.com/cmu-delphi/operations/blob/main/dev/docker/python/assets/setup.sh). I made some issues and PRs to remove at least some of them [here](https://github.com/cmu-delphi/operations/pull/70), but I figured we'll get there on low-priority speed.

Reply:
(Relatedly, I started a very long-term, low-priority project to redo the directory structure changes that `setup.sh` file makes to the Docker image [e.g. `mv repos/delphi/operations/src delphi/operations`]. That little change makes it so that all my IDE tools can't track imports at all and breaks my "Go To Reference" tools. Could be a worthwhile experiment.)

Reply:
> I wonder if those changes made in that setup script reflect the way the directory structure is on prod?

Yes -- they let the epidata acquisition system treat `delphi.epidata.acquisition` as a package without it having to be a valid (ie installable, with an `__init__.py` and `setup.py` and everything) Python package. Agree that this makes IDEs angry and angry IDEs make development needlessly difficult and error-prone, and that we should just make `delphi.epidata.acquisition` a package. Same with all other delphi "packages" that aren't really packages and only import because we bodge them on the filesystem like this.

Until we make that change, we need the CI and local dev environments to closely match production, so that we don't end up with changes that work in local dev but fail in prod.

Reply:
https://github.com/cmu-delphi/delphi-epidata/issues/939